common/djangoapps/student/helpers.py

```diff
@@ -12,6 +13,10 @@
 
 # Enumeration of per-course verification statuses
 # we display on the student dashboard.
+from openedx.core.djangoapps.user_api.accounts import PRIVATE_VISIBILITY, ACCOUNT_VISIBILITY_PREF_KEY, \
```

Comment:
Nit: use parenthesis to include a list of imports instead of wrapping with a `\`.